hangman_l.rb

```diff
@@ -0,0 +1,174 @@
+require "colorize"
+
+class Hangman
+
+  def initialize
+    @word_choices = ["cheeseburger", "cat", "dog", "whatever", "mississippi", "octothorpe", "splat", "bang"]
+    @answer = populate_answer
+    @wrong_counter = 0
+    @wrong_answers = []
+    @blank_answer_row = "_ " * @answer.length
+    @answer_output = @blank_answer_row.split(' ')
+
+  # Comment line below to show/hide answer
+    #puts @answer
+
+    puts @blank_answer_row
+    user_prompt
+  end
+
+  #randomly selects a word from the list
+  def populate_answer
+    random = @word_choices[rand(0...@word_choices.length)]
+    random = random.split(//)
+  end
+
+#--------------------------------------------------------
+# USER INPUT
+
+# Generates art/prompts for user interface
+  def user_prompt
+    # Compares user guess with answer
+    until @answer == @answer_output
+      print_art
+      puts "\nPlease enter a letter."
+      puts "\nWRONG GUESSES:" + @wrong_answers.join(" ") + "\n"
+      puts "YOUR GUESS:"
+      @guess = gets.chomp.downcase
```

Comment:
Setting `@guess` as an instance variable is fine, but since the user's guess is something that changes pretty frequently, it might make more sense to simply use a local variable. If you do that, what would you have to change about `check_user_input`?